### PR TITLE
RHDEVDOCS-4258 Distinguish between stopping support for v1alpha1 and removing the APIs

### PR DESCRIPTION
**Pull request by rolfedh**
- Aligned team: Dev Tools
- For branches: 
- Jira: https://issues.redhat.com/browse/RHDEVDOCS-4258
- Direct link to doc preview: https://rolfedh.github.io/previews/RHDEVDOCS-4258/cicd/pipelines/op-release-notes.html#deprecated-features-1-8_op-release-notes
- SME review: @siamaksade @vdemeester 
- QE review: @ppitonak
- Peer review: @ 	tbd
- <All reviews complete. Please merge now>

### CODE REVIEW

**Comment by rolfedh** [comment]
/assign @siamaksade 
WDYT?

File: modules/op-specifying-pipelines-resource-quota-using-priority-class.adoc [hunk 1]
```diff
@@ -167,7 +167,7 @@ Ensure that all tasks in the pipeline belongs to the same priority class.
 .Example: YAML for a pipeline run
 [source,yaml]
 ----
-apiVersion: tekton.dev/v1alpha1
+apiVersion: tekton.dev/v1beta1
```
Comment (ppitonak) [inline modules/op-specifying-pipelines-resource-quota-using-priority-class.adoc:170]: Both old and new version is missing workspace binding
```
spec:
  workspaces:
  - name: local-maven-repo
    emptyDir: {}
```

**Review by ppitonak — COMMENTED** [review]
@rolfedh I suggest to split this PR to two - one with text about deprecating/end of support and second one with code snippet changes. That way it will be easier to cherry-pick the code changes to older branches

File: modules/op-specifying-pipelines-resource-quota-using-priority-class.adoc [hunk 2]
```diff
@@ -130,7 +130,7 @@ spec:
 .Example: YAML for a task in the pipeline
 [source,yaml]
 ----
-apiVersion: tekton.dev/v1alpha1
+apiVersion: tekton.dev/v1beta1
```
[the remark this answers is not in the document]
Comment (vdemeester) [reply modules/op-specifying-pipelines-resource-quota-using-priority-class.adoc:133]: `failed quota` probably means there is a quota set in the namespace, and the specified value are not correct according to it 🙃 .

File: modules/op-release-notes-1-8.adoc [hunk 3]
```diff
@@ -26,14 +26,20 @@ include::snippets/technology-preview.adoc[]
...
+====
+For the *Tekton Operator*, the `operator.tekton.dev/v1alpha1` API version is *not* deprecated. You do not need to make changes to this value.
+====
 //  (link:https://github.com/tektoncd/triggers/pull/1103[#1103])
 
+
 * In {pipelines-title} 1.8, support for the `PipelineResource` custom resource (CR) has been removed. The `PipelineResource` CR was a Tech Preview feature and part of the `tekton.dev/v1alpha1` API.
 
-* In {pipelines-title} 1.8, support for the `Condition` custom resource (CR) has been removed. The `PipelineResource` CR was part of the `tekton.dev/v1alpha1` API.
+* In {pipelines-title} 1.8, support for the `Condition` custom resource (CR) has been removed. The `Condition` CR was part of the `tekton.dev/v1alpha1` API.
```
[the remark this answers is not in the document]
Comment (rolfedh) [reply modules/op-release-notes-1-8.adoc:42]: Updated. Thanks.

File: modules/op-specifying-pipelines-resource-quota-using-priority-class.adoc (hunk 2 above)
[the remark this answers is not in the document]
Comment (rolfedh) [reply modules/op-specifying-pipelines-resource-quota-using-priority-class.adoc:133]: Thanks! I've updated the first code block.

File: modules/op-specifying-pipelines-resource-quota-using-priority-class.adoc (hunk 1 above)
Comment (rolfedh) [reply modules/op-specifying-pipelines-resource-quota-using-priority-class.adoc:170]: Updated. Thanks.

**Pull request closed without merging**